packages/ses/test/test-anticipate-iterator-helpers-shimmed.js

```diff
@@ -0,0 +1,42 @@
+// KLUDGE HAZARD The core-js shims are written as sloppy code
+// and so introduce sloppy functions.
+import 'core-js/actual/iterator/index.js';
+import test from 'ava';
+import '../index.js';
+
+// KLUDGE HAZARD only for testing with the sloppy modules of the
+// core-js iterator shim.
+// We mutate the permits to tolerates the sloppy functions for testing
+// by sacrificing security. The caller and arguments properties of
+// sloppy functions violate ocap encapsulation rules.
```

Comment:
`core-js` use strict mode, but not in all cases - for example, in cases where `this` depends on it. `.arguments` and `.caller` are also interesting cases, I didn't think that it's interesting for anyone. I can consider adding the strict mode in all affected cases. You could open an issue in the `core-js` repo.

Reply:
https://github.com/zloirock/core-js/issues/1269

Thanks!